manifests/agent/params.pp

```diff
@@ -0,0 +1,24 @@
+# Calculate correct defaults for pipelines::agent parameters
+class pipelines::agent::params {
+  case $facts['kernel'] {
+    'Linux': {
+      $user_home = '/home/distelli'
+      $user_shell = '/bin/false'
+      $user_group = 'distelli'
+      $sudoers_path = '/etc/sudoers'
+    }
+    'Darwin': {
+      $user_home = '/Users/distelli'
+      $user_shell = '/usr/bin/false'
+      $user_group = 'staff'
+      $sudoers_path = '/private/etc/sudoers'
+    }
+    'windows': {
+      $user_home = 'C:/Users/distelli'
```

Comment:
Dangerous default.  Doesn't have to be `c:\`, or `C:\users`, or even `C:\users\Distelli`

Reply:
Sorry, I don't understand that. What should it be?

Reply:
So the issue is the system drive doesn't have to be `C:\`.  Also, the Users directory changes depending on which OS you're on.  And the directory your user uses at logon can change (name collisions or corrupt profiles will cause a different home directory to be used).

However, this is all moot.  After digging around the module code, the only place $user_home is used is during the user creation.  Which you don't (and shouldn't _really_) enforce during user creation anyway.

i.e. remove
https://github.com/puppetlabs/puppetlabs-distelli_agent/blob/2c63a9db4c7ca83eb84a4be50c4427ae24e757d2/manifests/deps/windows.pp#L30

and

https://github.com/puppetlabs/puppetlabs-distelli_agent/blob/2c63a9db4c7ca83eb84a4be50c4427ae24e757d2/manifests/deps/windows.pp#L33